reference/docs-conceptual/learn/remoting/WinRMSecurity.md

```diff
@@ -25,73 +27,94 @@ PowerShell Remoting (and WinRM) listen on the following ports:
+On private networks, the default Windows Firewall rule for PowerShell Remoting accepts all
+connections. On public networks, the default Windows Firewall rule allows PowerShell Remoting
+connections only from within the same subnet. You have to explicitly change that rule to open
+PowerShell Remoting to all connections on a public network.
 
->**Warning:** The firewall rule for public networks is meant to protect the computer from potentially malicious external connection attempts. Use caution when removing
->this rule.
+> [!Warning]
+> The firewall rule for public networks is meant to protect the computer from potentially malicious
+> external connection attempts. Use caution when removing this rule.
 
 ## Process isolation
 
-PowerShell Remoting uses [Windows Remote Management (WinRM)](https://msdn.microsoft.com/library/windows/desktop/aa384426) for communication between computers.
-WinRM runs as a service under the Network Service account, and spawns isolated processes running as user accounts to host PowerShell instances. An instance of PowerShell running as one
-user has no access to a process running an instance of PowerShell as another user.
+PowerShell Remoting uses [Windows Remote Management (WinRM)](/windows/win32/winrm/portal)
+for communication between computers. WinRM runs as a service under the Network Service account, and
+spawns isolated processes running as user accounts to host PowerShell instances. An instance of
+PowerShell running as one user has no access to a process running an instance of PowerShell as
+another user.
 
 ## Event logs generated by PowerShell Remoting
 
-FireEye has provided a good summary of the event logs and other security evidence generated by PowerShell Remoting sessions, available at
-[Investigating PowerShell Attacks](https://www.fireeye.com/content/dam/fireeye-www/global/en/solutions/pdfs/wp-lazanciyan-investigating-powershell-attacks.pdf).
+FireEye has provided a good summary of the event logs and other security evidence generated by
+PowerShell Remoting sessions, available at [Investigating PowerShell Attacks](https://www.fireeye.com/content/dam/fireeye-www/global/en/solutions/pdfs/wp-lazanciyan-investigating-powershell-attacks.pdf).
 
 ## Encryption and transport protocols
 
-It is helpful to consider the security of a PowerShell Remoting connection from two perspectives: initial authentication, and ongoing communication.
+It is helpful to consider the security of a PowerShell Remoting connection from two perspectives:
+initial authentication, and ongoing communication.
 
-Regardless of the transport protocol used (HTTP or HTTPS), PowerShell Remoting always encrypts all communication after initial authentication with a per-session AES-256 symmetric key.
+Regardless of the transport protocol used (HTTP or HTTPS), WinRM always encrypts all PowerShell remoting
```

Comment:
Isn't this what we are trying to avoid saying. It doesn't always encrypt the data, and even if it does the encryption could be poor/broken (see NTLM message encryption).